media/css/products/landing.scss

```diff
@@ -5,206 +5,60 @@
 $font-path: '/media/protocol/fonts';
 $image-path: '/media/protocol/img';
 
-@import '../protocol/components/custom-menu-list';
-@import '../protocol/components/fxa-form';
+@import '../protocol/components/picto';
```

Comment:
**Nit:** We changed how we import Protocol components through CSS sometime back in [Q1 of 2022](https://github.com/mozilla/bedrock/issues/11032). Instead of importing the component directly in a CSS file, we instead [created a universal import](https://github.com/mozilla/bedrock/blob/main/media/css/protocol/components/picto.scss) -- that includes all you'll need for that component -- that is then placed in [`static-bundles.json`](https://github.com/mozilla/bedrock/blob/main/media/static-bundles.json#L805), which we import within a `css_bundle` macro in the HTML page. More details here: https://bedrock.readthedocs.io/en/latest/coding.html#id2

So I would remove this line from this file and instead add `{{ css_bundle('protocol-picto')}}` in the `landing.html` HTML file

Reply:
Switched to using the universal import in the HTML. And removed two other universal imports that were no longer in use.